cea/demand/calibration/clustering/clustering_main.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+===========================
```

Comment:
@daren-thomas yes, but some of the case studies can become extensively big (when the optimisation is run it stores about 20GB of data.